README.md

```diff
@@ -1,2 +1,91 @@
 # test-automation
-Tools for making our tests easier to run
+
```

Comment:
Updated the README with more thorough instructions for connecting to the master and messing with ssh-agent